tap_github/schemas/issues.json

```diff
@@ -42,261 +39,240 @@
       }
     },
     "repository_url": {
-      "type": [
-        "null",
-        "string"
-      ]
+      "type": ["null", "string"]
     },
     "number": {
-      "type": [
-        "null",
-        "integer"
-      ]
+      "type": ["null", "integer"]
     },
     "closed_at": {
-      "type": [
-        "null",
-        "string"
-      ],
+      "type": ["null", "string"],
       "format": "date-time"
     },
     "labels_url": {
-      "type": [
-        "null",
-        "string"
-      ]
+      "type": ["null", "string"]
     },
     "title": {
-      "type": [
-        "null",
-        "string"
-      ]
+      "type": ["null", "string"]
     },
     "assignee": {
+      "$ref": "shared/user.json#/"
+    },
+    "assignees": {
       "type": [
```

Comment:
If you're going to keep type values on one line, please be consistent

Reply:
Updated to keep it as a single line